metaseq/cli/train.py

```diff
@@ -495,6 +495,57 @@ def post_checkpoint_callback(cfg, filename):
                 ),
             )
             os.remove(filename)
+
+            # Run evals after all checkpoint parts have been uploaded
+            checkpoint_id = checkpoint_dir.split("_")[-1]
```

Comment:
Could we move this logic to a separate method and delete the existing `_run_evaluations` if it's not used anymore?